app/facades/food_facade.rb

```diff
@@ -0,0 +1,15 @@
+class FoodFacade
+
+  def self.service(food)
+    FoodService.get_details(food)
+  end
+
+  def self.food_info(info)
+    food = service(info)
+
+    food.map do |food_data|
+      Food.new(food_data)
+    end
+  end
```

Comment:
This looks mostly good, but we don't have a way to extract the `totalResults` here, which is the responsibility of the facade.